src/PharoThings-Devices-HD44780.package/PotLCD.class/instance/write.charMode..st

```diff
@@ -0,0 +1,15 @@
+writing
+write: anHex charMode: aBit
+	| j |
+	j := 1.
+	(Delay forSeconds: 0.0001) wait.
+	"1 miliseconds waiting to prevent write too quickly"
+	(gpioDic at: #modePin) value: aBit.
+	8 to: 1 by: -1 do: [ :i | 
+		(busMode = '#4bit' and: [ j > 4 ]) 
```

Comment:
Maybe busMode can be a number of bits like busBitSize. It would equal 4 for this case of #'4bit' and code will be simpler:
```Smalltalk
        j > busBitSize ifTrue: [ j:= 1 ... 
```

Reply:
Nice! :)

Reply:
And why we really need this variable if it is in fact "dataPins size"?
It can be 
```Smalltalk
j > dataPins size ifTrue: [ j:= 1 ... 
```

Reply:
Nice nice!